wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -68,10 +68,19 @@ def get_site_by_code(self, site_code):
         try:
             s = self.db_session.query(odm2_models.Sites). \
                 filter(odm2_models.Sites.SamplingFeatureCode == site_code).one()
+
+            aff = self.db_session.query(odm2_models.Affiliations). \
+                join(odm2_models.ActionBy). \
+                join(odm2_models.Actions). \
+                join(odm2_models.FeatureActions). \
+                join(odm2_models.Sites). \
+                filter(odm2_models.Sites.SamplingFeatureCode == s.SamplingFeatureCode). \
+                filter(odm2_models.ActionBy.IsActionLead == True).first()
```

Comment:
The criteria `odm2_models.ActionBy.IsActionLead == True` may be too strict, in this context. Either remove it (that *could* lead to problems in some situations), or add a comment next to the line in the code, for future reference.